static_src/actions/user_actions.js

```diff
@@ -26,9 +26,9 @@ const userActions = {
     });
   },
 
-  fetchSpaceUsers(spaceGuid) {
+  fetchSpaceUserRoles(spaceGuid) {
```

Comment:
This is great. I was confused as to the naming convention broke from the tradition here.